merge_reports.js

```diff
@@ -3,27 +3,44 @@
 
 const { config } = require('./config')
 
+/**
+ * Creates the final summary based on how many errors, warnings and notices there is
+ * so the final summary to be meaningful.
+ * @param {Number} errors the number of errors found
+ * @param {Number} warnings the number of warnings found
+ * @param {Number} notices the number of notices found
+ */
+function createFinalSummaryMessage (errors, warnings, notices) {
+  let summary = ''
+
+  const errorsMessage = errors > 1 ? ':x: ' + errors + ' errors\n'
```

Comment:
```suggestion
  const errorsMessage = `:x: ${errors} errors\n`
```

Reply:
If you use `:x: ${errors} errors\n` only you will have a situation when the output will be:
`1 errors` which we don't want.